Content/Items/Dungeon/Weapons.Cloudstrike.cs

```diff
@@ -210,6 +210,8 @@ public class CloudstrikeShot : ModProjectile, IDrawAdditive, IDrawPrimitive
 		private Vector2 oldPlayerPos = Vector2.Zero; //These 2 variables are used for following the Player
 		private float oldRotation = 0f;
 
+		public NPC host = default;
```

Comment:
We should document why this is here -- im assuming its because the enemy re-uses this projectile? We should make note of that in this projectile's class so we can track down this behavior later. It may also potentially be useful to pull out a copy of this as it's own projectile as I can forsee wanting to make diverging changes for balance's sake in the future,

Reply:
It's only used for the visual sparks around the enemy. But I added a comment to it